.github/workflows/example-ios-build-check.yml

```diff
@@ -45,29 +45,22 @@ jobs:
       - name: Check out Git repository
         uses: actions/checkout@v4
 
-      - name: Install monorepo node dependencies
-        run: yarn install
-      - name: Restore Reanimated node_modules from cache
+      # Does this optimization even work with new yarn?
+      # Perhaps we should cache `yarn's` cache instead?
```

Comment:
I think we should stay consistent with these comments and stick with `TODO` convention:

`# TODO: Check if this optimization works with new yarn`